app/controllers/pages_controller.rb

```diff
@@ -1,25 +1,20 @@
 class PagesController < ApplicationController
-  def home
-  end
-
-  def about
-  end
-
-  def resources
+  def adopt
+    @available_rats = available_rats
   end
 
-  def contact
-  end
+  private
 
-  def volunteer
+  def petfinder
+    @_petfinder ||= Petfinder::Client.new
   end
 
-  def adopt
+  def any_rat_rescue
+    @_any_rat_rescue ||= petfinder.shelter("AZ173")
```

Comment:
The string `"AZ173"` here is what we call a "Mystery Guest". Other developers have no real way of knowing where this came from or why it exists. A typical solution to this problem is to extract a `CONSTANT` out to give it a name. What would you think about extracting a constant called `ANY_RAT_RESCUE_PETFINDER_ID` at the top of this controller and using it here?